openhcl/openhcl_boot/src/main.rs

```diff
@@ -123,8 +123,12 @@ fn build_kernel_command_line(
         "panic_print=0",
         // RELIABILITY: Reboot immediately on panic, no timeout.
         "panic=-1",
-        // RELIABILITY: Print processor context information on a fatal signal.
-        "print_fatal_signals=1",
+        // RELIABILITY: Don't print processor context information on a fatal
+        // signal. Our crash dump collection infrastructure seems reliable, and
+        // this information doesn't seem useful without a dump anyways.
+        // Additionally it may push important logs off the end of the kmsg
+        // page logged by the host.
+        //"print_fatal_signals=0",
```

Comment:
Hm, what do the last few lines look like on panic now? Recall two things:
* Recent investigations have shown a SIGABRT line + processor context, but no usermode panic info, and
* Isolated VMs' data is protected; hosts don't get to see the crash info.

Perhaps this should be '1' for isolated VMs?

Reply:
The goal of this change is to reduce the logging below the rust panic output so that we do get that usermode panic info. That's far more useful than register values. That doesn't really change if we're isolated or not.